npm install fails when husky is not present in the workspace

seems that the postInstall hook requires husky to exist in the `node_modules/` however it is only defined as a devDependency, so it does not get pulled at install time.

```
npm ERR! code 127
npm ERR! path /path/to/project/node_modules/path-equal
npm ERR! command failed
npm ERR! command sh /tmp/postinstall-3826db94.sh
npm ERR! /tmp/postinstall-3826db94.sh: line 1: husky: command not found

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/rajiteh/.npm/_logs/2022-10-28T02_21_29_799Z-debug-0.log
```

--- a/package.json
+++ b/package.json
@@ -40,7 +40,7 @@
     "depcheck": "depcheck",
     "lint": "eslint --ext=ts,js .",
     "nuke": "yarn clean && rimraf node_modules",
-    "postinstall": "husky install",
+    "prepare": "husky install",
     "prepack": "pinst --disable",
     "postpack": "pinst --enable",
     "size": "size-limit",

